src/grammar.lalrpop

```diff
@@ -16,11 +16,12 @@ use crate::term::{BinaryOp, RichTerm, Term, UnaryOp, StrChunk, MetaValue,
     MergePriority, Contract, NAryOp, RecordAttrs};
 use crate::term::make as mk_term;
 use crate::types::{Types, AbsType};
+use crate::parser::utils::mk_span;
```

Comment:
Nitpick: there's already a lot of `parser::utils::{...}` imports just above, we could put `mk_span` with the others.